packages/kbn-coloring/src/shared_components/color_mapping/color/color_handling.test.ts

```diff
@@ -343,30 +319,30 @@ describe('Color mapping - color generation', () => {
           steps: [
             {
               type: 'categorical',
-              paletteId: EUIAmsterdamColorBlindPalette.id,
+              paletteId: KbnPalette.Default,
               colorIndex: 0,
               touched: false,
             },
             {
               type: 'categorical',
-              paletteId: EUIAmsterdamColorBlindPalette.id,
+              paletteId: KbnPalette.Default,
               colorIndex: 2,
               touched: false,
             },
           ],
           sort: 'desc',
         },
       },
-      getPaletteFn,
+      palettes,
       false,
       {
         type: 'categories',
         categories: ['cat1', 'cat2', 'cat3'],
       }
     );
-    expect(toHex(colorFactory('cat1'))).toBe(toHex(EUI_AMSTERDAM_PALETTE_COLORS[0])); // EUI green
+    expect(toHex(colorFactory('cat1'))).toBe(toHex(elasticPaletteColors[0])); // EUI green
     expect(toHex(colorFactory('cat2'))).toBe('#a4908f'); // red gray green
-    expect(toHex(colorFactory('cat3'))).toBe(toHex(EUI_AMSTERDAM_PALETTE_COLORS[2])); // EUI pink
+    expect(toHex(colorFactory('cat3'))).toBe(toHex(elasticPaletteColors[2])); // EUI pink
```

Comment:
Fixed in 016f1615074a66354d91174004f5b9ea31ef4a56. It would be nicer to access the color directly like `euiVars.pinkX` than a comment. But I don't think the vis colors easily accessible atm.